users/updates/components/86FkFVV4TGRg.php

```diff
@@ -7,9 +7,9 @@
 $countE=0;
 
 $db->query("ALTER TABLE groups_menus
-  MODIFY COLUMN id int(11) UNSIGNED NOT NULL AUTO_INCREMENT,
-  MODIFY COLUMN group_id int(11) UNSIGNED NOT NULL,
-  MODIFY COLUMN menu_id int(11) UNSIGNED NOT NULL");
+  ALTER COLUMN id SET DATA TYPE SERIAL,
+  ALTER COLUMN group_id SET DATA TYPE INTEGER,
```

Comment:
Please add `NOT NULL` constraint as well, if not already present.

Reply:
I must have forgotten I added to some other ones I missed too.